Sources/RingPublishingTracking/Public/Model/Audio/AudioMetadata.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AudioMetadata.swift
+//  RingPublishingTracking
+//
+//  Created by Bernard Bijoch on 08/10/2024.
+//  Copyright © 2024 Ringier Axel Springer Tech. All rights reserved.
+//
+
+import Foundation
+
+/// Audio metadata for event tracking
+public struct AudioMetadata: Encodable {
+
+    /// Content identifier in source system (CMS)
+    public let contentId: String
+
+    /// Content title in source system
+    public let contentTitle: String
+
+    /// Content series identifier
+    public let contentSeriesId: String?
+
+    /// Content series title
+    public let contentSeriesTitle: String?
+
+    /// Audio media type (tts / podcast / livestream etc.)
+    public let mediaType: String
+
+    /// Audio duration (in seconds)
+    public let audioDuration: Int?
+
+    /// Type of audio stream format loaded into player
+    public let audioStreamFormat: AudioStreamFormat
+
+    /// Is audio a part of a bigger content piece
+    public let isContentFragment: Bool
+
+    /// Audio content category in terms of paid access to it
+    public let audioContentCategory: AudioContentCategory
+
+    /// Audio player version name, for example 1.2.0
+    public let audioPlayerVersion: String
+
+    public var audioEventParameter: AudioEventParameter {
```

Comment:
czemu to tutaj jest? ten strukt może być prywatny i go sobie dalej uzupełnisz?

Reply:
Też przeniesione do private i do extensionów